### DISCUSSION TITLE
Handling of bind mounts?

### DISCUSSION THREAD

**Discussion by eitsupi** [devcontainers/community#77] [Q&A]
Dev Container CLI bind mounts workspaces by default, but I am wondering if it specifies something like `cached` by default (unfortunately I could not understand it from reading the source code).
This is pretty important for users using non-Linux filesystems (i.e. often macOS users), so it may be great if it was mentioned in the spec.

**Comment by samruddhikhandale** [comment]
Looping in @chrmarti for his expertise in this area.
